pkg/logs/message/message.go

```diff
@@ -5,25 +5,46 @@
 
 package message
 
+import "github.com/DataDog/datadog-agent/pkg/logs/config"
+
 // Message represents a log line sent to datadog, with its metadata
 type Message struct {
 	Content    []byte
 	Origin     *Origin
-	status     string
-	Timestamp  string
 	RawDataLen int
+	Timestamp  string
+
+	status string
+}
+
+// NewPartialMessage3 constructs message with partial information.
```

Comment:
Might be good to explain the differences between NewPartialMessage 1/2/3 (and maybe name them accordingly)

Reply:
I think the function self-explains which fields are passed in. It might be duplicated to put those info again as name. But I will put better comments.